sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -73,7 +74,8 @@ class RocksDB(
     loggingId: String = "",
     useColumnFamilies: Boolean = false,
     enableStateStoreCheckpointIds: Boolean = false,
-    partitionId: Int = 0) extends Logging {
+    partitionId: Int = 0,
+    eventListener: Option[RocksDBEventListener] = None) extends Logging {
```

Comment:
This is `None` only for unit tests right ?

Reply:
Yeah, just for a few tests in RocksDBSuite that create this without the provider initialization